src/main/java/org/usfirst/frc4904/auton/PickupAndShootSide.java

```diff
@@ -0,0 +1,32 @@
+package org.usfirst.frc4904.auton;
+
+import java.util.Arrays;
+
+import org.usfirst.frc4904.robot.RobotMap;
+import org.usfirst.frc4904.robot.commands.FlywheelSpinDown;
+import org.usfirst.frc4904.robot.commands.RunIntake;
+import org.usfirst.frc4904.robot.commands.Shoot;
+import org.usfirst.frc4904.robot.commands.vision.VisionMoveHighPort;
+import org.usfirst.frc4904.standard.commands.chassis.SimpleSplines;
+
+import edu.wpi.first.wpilibj.trajectory.Trajectory;
+
+class PickupAndShootSide extends AutonRoutine {
+
+        public PickupAndShootSide() {
+                Trajectory goingToPowerCells = RobotMap.Component.sensorChassis
+                                .generateQuinticTrajectory(Arrays.asList(Poses.currentPos, Poses.sideCollectStart));
+                SimpleSplines approachSpline = new SimpleSplines(RobotMap.Component.sensorChassis, goingToPowerCells);
+                this.andThen(approachSpline);
+                this.andThen(new RunIntake(RobotMap.Component.intake));
+                Trajectory collect = RobotMap.Component.sensorChassis
+                                .generateQuinticTrajectory(Arrays.asList(Poses.sideCollectStart, Poses.sideCollectEnd));
+                SimpleSplines collectSpline = new SimpleSplines(RobotMap.Component.sensorChassis, collect);
+                this.andThen(collectSpline);
+                andThen(new VisionMoveHighPort());
+                double FlywheelSpeed = 0.0;
+                andThen(new Shoot(RobotMap.Component.indexer, RobotMap.Component.shooter, FlywheelSpeed));
```

Comment:
if you move to the high port, use the `BoxShot` command instead of `Shoot`

Reply:
`AutonRoutine` is used as a wrapper for the spline visualizer to access auton commands